Add docs for HTTP `Permissions-Policy` header `cross-origin-isolated` directive (#36780)
mdn/content: files/en-us/web/http/headers/permissions-policy/index.md

  - : Controls access to the [Compute Pressure API](/en-US/docs/Web/API/Compute_Pressure_API).

- {{httpheader('Permissions-Policy/cross-origin-isolated','cross-origin-isolated')}} {{Experimental_Inline}}

>   - : Controls whether the current document is allowed to be cross-origin isolated.

Reviewer: This, but with link to cross origin isolated definition.
```suggestion
  - : Controls whether the document is allowed to use APIs that are restricted to cross-origin isolated documents.
```

>   - : Controls whether the current document is allowed to be {{domxref("Window.crossOriginIsolated", "cross-origin isolated", "", 1)}}.

Reviewer: ```suggestion
  - : Controls whether the current document is treated as {{domxref("Window.crossOriginIsolated", "cross-origin isolated", "", 1)}}.
```

  - : Controls access to the [Compute Pressure API](/en-US/docs/Web/API/Compute_Pressure_API).

- {{httpheader('Permissions-Policy/cross-origin-isolated','cross-origin-isolated')}} {{Experimental_Inline}}

>   - : Controls whether the current document is treated as {{domxref("Window.crossOriginIsolated", "cross-origin isolated", "", 1)}}.

Author: ```suggestion
  - : Controls whether the current document can be treated as {{domxref("Window.crossOriginIsolated", "cross-origin isolated", "", 1)}}.
```
